eng/pipelines/aggregate-reports.yml

```diff
@@ -156,4 +156,46 @@ stages:
 
           - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
             displayName: 'Publish Security Analysis Logs'
+            condition: succeededOrFailed()
+
+      - job: APIScan
+        timeoutInMinutes: 180
+        steps:
+          - template: /eng/pipelines/templates/steps/install-dotnet.yml
+
+          - task: PowerShell@2
+            displayName: Download Latest Dev Packages
+            inputs:
+              pwsh: true
+              filePath: 'eng/scripts/DownloadDevPackages.ps1'
+              arguments: >
+                -WorkingDirectory '$(Build.ArtifactStagingDirectory)'
+                -NupkgFilesDestination 'nupkgFiles'
+
+          - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@2
+            displayName: 'Run APIScan'
+            inputs:
+              softwareFolder: $(Build.ArtifactStagingDirectory)
+              # SoftwareName cannot change as it is used for looking up baselining data
+              softwareName: 'Azure SDK Libraries'
+              softwareVersionNum: '$(OfficialBuildId)'
+              isLargeApp: false
+              preserveLogsFolder: true
+              verbosityLevel: standard
+            env:
+              # azure-sdk-apiscan (81109e5f-0620-423c-a37a-c22fbf8973a7)
+              AzureServicesAuthConnectionString: runAs=App;AppId=81109e5f-0620-423c-a37a-c22fbf8973a7;TenantId=72f988bf-86f1-41af-91ab-2d7cd011db47;AppKey=$(azure-sdk-apiscan-client-secret)
```

Comment:
Would it be easier to wrap the whole conn string into the secret? That way if we change the app we don't have to update it in two places.

Reply:
I thought about that, and I was trying to keep just the secret as opposed to a full configuration. I think of this as no different than a username which we generally don't put into KV.